src/safeposix/syscalls/fs_calls.rs

```diff
@@ -7,6 +7,7 @@ use crate::safeposix::filesystem::*;
 use crate::safeposix::net::{NET_METADATA};
 use crate::safeposix::shm::*;
 use super::fs_constants::*;
+use std::convert::TryInto;
```

Comment:
do you still use this? if so we need to move this into the interface files, i think it may have been exported already?